package.json

```diff
@@ -110,6 +110,7 @@
     "jest-watch-typeahead": "^0.6.5",
     "jsonfile": "^6.0.1",
     "lodash": "^4.17.21",
+    "mime": "^3.0.0",
```

Comment:
This change was needed so that the tests are running the same version of `mime` as `@remix-run/deno`. Without this change, the Deno tests ended up using a different version of `mime` (hoisted from some of our dependencies) which requires Node's `path` and `fs` modules. Since we're no longer providing Node.js polyfills by default, this breaks the build.